pandas/core/categorical.py

```diff
@@ -409,6 +410,11 @@ def dtype(self):
         """The :class:`~pandas.api.types.CategoricalDtype` for this instance"""
         return self._dtype
 
+    @property
+    def _block_type(self):
```

Comment:
I'm still thinking about how best to handle this. The conflict is

1. We don't want 3rd party libs to worry about `Blocks` at all
2. We (maybe) want Categorical, etc. to be instances of `ExtensionArray`
3. We *do* want to use `CategoricalBlock` instead of the default `ExtensionBlock`

Reply:
Yeah, I'm looking through at the methods `CategoricalBlock` implements, that aren't part of the interface

1. `fillna`: should probably be added
2.  `interpolate`: Actual interpolation probably isn't important enough, but I think this is called indirectly for `fillna`?
3. `shift`
4. `to_native_types`

Trying to balance complexity of implementation for 3rd-parties here.